crates/sui-framework/sources/governance/sui_system.move

```diff
@@ -364,22 +367,34 @@ module sui::sui_system {
         let delegator_reward = balance::split(&mut computation_reward, delegator_reward_amount);
         balance::join(&mut self.storage_fund, storage_reward);
 
+        let storage_fund_reward_amount = storage_fund * computation_charge / total_stake;
```

Comment:
risk of overflow?

Reply:
Good catch. This and reward amount calculation for delegators above can both overflow. Will update